src/main/java/chalmers/pimp/controller/components/LayerItemPane.java

```diff
@@ -87,4 +130,25 @@ private void updateVisibilityImage() {
       imageView.setImage(EYE_CLOSED_IMAGE);
     }
   }
+
+  /**
+   * Sets the style for this {@code LayerItemPane} to Gray if the corresponding Layer is active.
+   * Reverts to CSS default otherwise.
+   */
+  private void showIfLayerIsSelected() {
+    if (layer == model.getActiveLayer()) {
+      this.setStyle("-fx-background-color: GREY;");
```

Comment:
I think it's possible to use constants defined in our own stylesheet here. We have the -selected-color selector for this purpose.